src/indexer/merger.rs

```diff
@@ -303,6 +322,82 @@ impl IndexMerger {
         Ok(())
     }
 
+    // used to merge `u128` single fast fields.
+    fn write_u128_multi_fast_field(
+        &self,
+        field: Field,
+        fast_field_serializer: &mut CompositeFastFieldSerializer,
+        doc_id_mapping: &SegmentDocIdMapping,
+    ) -> crate::Result<()> {
+        let segment_and_ff_readers = self
```

Comment:
The type is non trivial.
```suggestion
        let segment_and_ff_readers: Vec<(&SegmentReader, MultivaluedU128FastFieldReader<u128>>)> = self
```